solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/RankSVMModel.java

```diff
@@ -80,7 +79,7 @@ public Explanation explain(LeafReaderContext context, int doc,
     for (final Explanation featureExplain : featureExplanations) {
       final List<Explanation> featureDetails = new ArrayList<>();
       featureDetails.add(Explanation.match(featureToWeight[index],
-          "weight on feature [would be cool to have the name :)]"));
```

Comment:
The explain includes the name now as a part of the feature explain

> 
7.0 = RankSVMModel(name=svm-efi) model applied to features, sum of:
&nbsp;&nbsp;&nbsp;&nbsp;5.0 = prod of:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.0 = weight on feature
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5.0 = ValueFeature [name=sampleConstantA, params={value=5}]
&nbsp;&nbsp;&nbsp;&nbsp;2.0 = prod of:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2.0 = weight on feature
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.0 = ValueFeature [name=sampleConstantB, params={value=1}]